docs/prometheus-exporter.md

```diff
@@ -116,3 +106,19 @@ The block height of the last arrived block.
 Timestamp for the processed last arrived block (as Unix time in milliseconds).
 
 Note that this is the time when the node has last processed a new block. It is **not** the objective timestamp of the block (i.e., slot time).
+
+### `consensus_received_messages_total`
```

Comment:
I don't think adding total is providing more information, and is just here because the Prometheus documentation recommends having it in this case.

Also changing this now, would make it inconsistent with the other metrics like this, so I'll leave it as is